permissions/plugin-puppet-enterprise.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "puppet-enterprise-pipeline"
+paths:
+- "org/puppet/plugins/puppet-enterprise-pipeline"
```

Comment:
According to the pom.xml, your groupId is `org.jenkins-ci.plugins.workflow`, so this should be `org/jenkins-ci/plugins/workflow/puppet-enterprise-pipeline`.